pkg/resource/db_instance/hooks.go

```diff
@@ -90,6 +90,13 @@ var (
 		errors.New("DB instance in 'deleting' state, cannot be modified or deleted."),
 		ackrequeue.DefaultRequeueAfterDuration,
 	)
+
+	customWaitAtferUpdate = ackrequeue.NeededAfter(
+		errors.New("ModifyDBInstance API was successful but it"+
+			" asynchronously updates the DBInstanceStatus. Requeue to find the"+
+			" current DBInstance status"),
```

Comment:
Please change the message to inform the user that they should not take any action. The imperative "Requeue to find the current DBInstance status" makes it sound like the user should take an action...

Suggest rewording to this:

`Requeueing resource after successful update; status fields retrieved asynchronously."